fixed-hash/Cargo.toml

```diff
@@ -1,23 +1,24 @@
 [package]
 name = "fixed-hash"
-version = "0.4.0"
+version = "0.4.1"
 authors = ["Parity Technologies <admin@parity.io>"]
 license = "MIT"
 homepage = "https://github.com/paritytech/parity-common"
 repository = "https://github.com/paritytech/parity-common"
 description = "Macros to define custom fixed-size hash types"
 documentation = "https://docs.rs/fixed-hash/"
 readme = "README.md"
+edition = "2018"
 
 [package.metadata.docs.rs]
 features = ["quickcheck", "api-dummy"]
 
 [dependencies]
-rand = { version = "0.5", optional = true, default-features = false }
+rand = { version = "0.6", optional = true, default-features = false }
```

Comment:
I meant any breaking upgrade in dependencies is a breaking change, since they are used in public API.
So I would suggest to upgrade to latest dependencies, but revert the version bump, so we can publish it after #217 is merged.